winterfell/webapp/src/templates/claimSelectForms.jade

```diff
@@ -1,15 +1,34 @@
 #select-forms-view
   vfi-breadcrumbs(page="1")
-  h2.header-wrapper Select Forms to Fill
-  div.vfi-form-card(ng-repeat="(formId, form) in forms", ng-class="{'vfi-required': form.required, 'vfi-completed': form.completed}")
-    h3.vfi-form-title {{form.name}}
-    h5.vfi-form-description {{form.description}}
-    a.vfi-form-btn(ui-sref="root.form({claimId: claimId, formId: formId})", ng-if="!form.completed")
-      div.vfi-icon-wrapper
-        div.vfi-next-icon
-    div.vfi-icon-wrapper.vfi-completed(ng-if="form.completed")
-      div.vfi-completed-icon
-  div.footer-wrapper
-    div.vfi-button-wrapper
-      button.vfi-cancel-btn(ng-click="onClickCancel()") Cancel
-      button.vfi-finish-btn(ui-sref="root.claimconfirm({claimId: claimId})") Finish
+  #select-forms-wrapper
+    h3 Select Forms for Claim
+
+    div.section.required
+      div.section-header
+        h4 Required Forms:
+        uib-progressbar.progressbar(value="numRequiredCompleted", max="requiredForms.length", type="{{progressBarType}}")
```

Comment:
Just a note im eventually going to rig this up to show the percent completion in terms of questions answered, and not just in terms of forms completed.